cypress/support/commands/common.js

```diff
@@ -442,7 +442,10 @@ Cypress.Commands.add('c_loadingCheck', () => {
 Cypress.Commands.add('c_authorizeCall', () => {
   try {
     const oAuthNewToken = Cypress.env('oAuthToken')
-    cy.task('authorizeCallTask', oAuthNewToken)
+    cy.task('authorizeCallTask', oAuthNewToken).then((response) => {
+      const email = response
+      Cypress.env('actualEmail', email)
```

Comment:
Why not ? Will it not work?
```suggestion
      Cypress.env('actualEmail', response)
```